_source/api/logzio-public-api.yml

```diff
@@ -2701,6 +2705,50 @@ paths:
         - lang: cURL
           source: 'curl -X POST "https://api.logz.io/v1/shared-tokens" -H  "accept: application/json" -H  "content-type: application/json" -H  "X-API-TOKEN: <token>" -d "{ \"tokenName\": \"Support shared token\", \"filters\":[339]}"'
 
+  # ::::: API TOKENS
+  /v1/api-tokens/sub-account:
+    post: 
+      operationId: createSubaccountAPIToken
+      summary: Create a subaccount API token
+      description: >-
+        Creates a new API token for a subaccount.
+
+      tags:
+        - Manage API tokens
+      parameters: 
+        - in: body
+          name: body
+          required: true  
+          schema:
+            $ref: '#/definitions/CreateSubaccountAPITokenRequest'
+      responses:
+        200:
+          description: successful operation
+          schema:
+            $ref: '#/definitions/SubaccountAPIToken'
+        400:
+          description: not found
+          schema:
+            type: object
+            properties:
+              message:
+                type: string
+                example: API token with id 7386 not found for account 160343
+                description: The API token or query filter could not be found
+        403:
+          description: forbidden
+          schema:
+            type: object
+            properties:
+              message:
+                type: string
+                example: Insufficient privileges
+                description: Insufficient privileges. Contact our Support team for access to this API feature.
+      x-code-samples:
+        - lang: cURL
+          source: 'curl -X POST "https://api.logz.io/v1/subaccount-api-token" -H  "accept: application/json" -H  "content-type: application/json" -H  "X-API-TOKEN: <token>" -d "{ \"tokenName\": \"newAPITokenTest999\", \"accountId\":7386}"'
```

Comment:
this curl will never work, as the path is wrong. it should be /v1/api-tokens/sub-account as stated in line 2709. @yuvalschneider better check these example before we publish these.

Reply:
updated